src/www/ui_components/add-server-view.js

```diff
@@ -230,6 +230,8 @@ Polymer({
   },
 
   ready: function() {
+    this.fire('ClipboardTextRequested');
```

Comment:
Why is this event needed? We already have a callback when the window is opened, I think

Reply:
I hadn't tested it yet. But sure.